packages/esm-patient-chart-app/src/clinical-views/types.ts

```diff
@@ -7,7 +7,7 @@ export type TableHeaderType = {
 };
 
 export interface Encounter extends OpenmrsResource {
-  encounterDatetime: Date;
+  encounterDatetime: string;
```

Comment:
Because this gets actually set to an ISO string and not a Date object

Reply:
Should we extend the `EncounterPropertyType` enum to include `encounterDatetime'?

Reply:
Yeah, thank you for catching that!